examples/cc3200/xively_firmware_updates/main.c

```diff
@@ -983,7 +982,7 @@ void SimpleLinkNetAppEventHandler( SlNetAppEvent_t* pNetAppEvent )
 
             SET_STATUS_BIT( g_ulStatus, STATUS_BIT_IP_AQUIRED );
 
-            /* Ip Acquired Event Data *.
+            /* Ip Acquired Event Data */
```

Comment:
I don't think it is a fault. It is more like a great experience how things can go wrong in code. So this remained a valid (compiler accepted this) C code. Miracle.